source/common/config/utility.h

```diff
@@ -219,6 +219,20 @@ class Utility {
     return config;
   }
 
+  template <class ProtoMessage, class Factory>
+  static ProtobufTypes::MessagePtr translateToFactoryRDSConfig(const ProtoMessage& source,
+                                                               Factory& factory) {
+    ProtobufTypes::MessagePtr config = factory.createEmptyRDSConfigProto();
+
+    if (config == nullptr) {
```

Comment:
I would just RELEASE_ASSERT() this like I recently did for the other instances of this.

Reply:
I was being consistent with the translate method above. Shall I update both?

Reply:
Yes please. I don't think it's really needed to throw an exception here.